examples/weight-init/README.md

```diff
@@ -0,0 +1,93 @@
+# Weight Initialization
+
+Table of Contents
+=================
+* [Introduction](#introduction)
+* [Selecting a weight initialization strategy](#selecting-a-weight-initialization-strategy)
+* [Experiments with LeNet on MNIST](#experiments-with-lenet-on-mnist)
+
+## Introduction
+
+Weight initialization can critically affect the speed at which a neural network is able to learn. Under certain circumstances,
```

Comment:
Sure! I did a little test to try it out myself:
https://github.com/lukeyeager/github-testing/blob/2d2fc50ff9/CONTRIBUTING.md